Automattic/newspack-plugin
# feat: add Newspack Elections block patterns

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds the Newspack Elections patterns to the Newspack Plugin.

### How to test the changes in this Pull Request:

1. Apply this PR and install the [Newspack Elections](https://github.com/Automattic/newspack-elections) plugin.
2. Create a page.
3. Navigate to the Block Insert menu (`+` button in top left header) > Patterns, and confirm that there's a section for Newspack Elections, with six patterns.
4. Insert each of the six patterns one at a time, and confirm:
   * That the contents of the pattern matches the examples under WP Admin > Posts > Patterns on https://ras-acc.newspackstaging.com.
   * That all blocks render correctly -- specifically homepage post blocks, subscription blocks, and donate blocks.
5. Deactivate the Newspack Elections plugin.
6. Navigate to the Block Insert menu (`+` button in top left header) > Patterns and confirm that the Elections section is now gone.


### Other information:
